templates/core/.env.sample

```diff
@@ -5,6 +5,8 @@ TRE_ADDRESS_SPACE="10.0.0.0/12"
 DEPLOY_GITEA=true
 DEPLOY_NEXUS=true
 RESOURCE_PROCESSOR_TYPE="vmss_porter"
+API_APP_SERVICE_PLAN_SKU_SIZE="P1v2"
```

Comment:
are these actually used? can't see where?

Reply:
The default is set in the Terraform as P1v3. Setting this in the env file basically means the developers of TRE will use a SKU that does not encounter Quota issues. The subscription that the build works in has quota setup